pilot/pkg/security/authz/builder/extauthz.go

```diff
@@ -100,9 +100,7 @@ func processExtensionProvider(in *plugin.InputParams) map[string]*builtExtAuthz
 		resolved[config.Name] = parsed
 	}
 
-	if authzLog.DebugEnabled() {
-		authzLog.Debugf("Resolved extension providers: %v", spew.Sdump(resolved))
-	}
+	authzLog.Debugf("Resolved extension providers: %v", spew.Sdump(resolved))
```

Comment:
IIUC, this change will make `spew.Sdump(resolved)` run first

Reply:
done